src/stitching/mergeSchemas.ts

```diff
@@ -122,6 +140,11 @@ function mergeSchemasImplementation({
         });
       }
 
+      const directiveInstances = schema.getDirectives();
+      directiveInstances.forEach(directive => {
```

Comment:
Hey @freiksenet what do you think about making the `if (mergeDirectives)` check here and [here](https://github.com/apollographql/graphql-tools/pull/1003/files#diff-bb87a5c1cf8e770e2b377a7e61904637R173) and then unconditionally passing `directives` to `new GraphQLSchema`?